extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -608,7 +608,6 @@ const transactionSubmissionSlice = createSlice({
     resetSubmission: () => initialState,
     resetAccountBalanceStatus: (state) => {
       state.accountBalanceStatus = initialState.accountBalanceStatus;
-      state.accountBalances = initialState.accountBalances;
```

Comment:
This is a bug from the fix to the loading glitch in AccountList.
We can't actually reset balances here(and don't need to really) because not all other pages fetch the account balances before using them.